### PR TITLE
Fixing typos

### PR DESCRIPTION
**Pull request by honzajavorek**
:memo: :heavy_check_mark: :x:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,10 +12,10 @@ This library takes API Blueprint AST and returns specific HTTP transactions (Req
 - Inherits parameters from parent Resource and Action sections
 - Expands URI templates
 - Warns on undefined URI query and path parameters
-- Validates URI parameteres types
-- Selects first request and first response if multiple request or responses are given in the 
-- Assigns path origin object pointing to the Blueprint AST 
-- Compiles [cannonical transaction path](#canonical-transaction-paths) as a unique identifier for each transaction
+- Validates URI parameters types
+- Selects first request and first response if multiple request or responses are given in the
```
Comment (kylef) [inline README.md]: This still doesn't make sense.

"if multiple request or responses are given in the..." it's missing some word after the? (I don't know what it is)

Comment (honzajavorek) [inline README.md]: Yes, I did not see this. There should be probably "Blueprint AST".

Comment (honzajavorek) [inline README.md]: Fixed (amended commit)

File: README.md [hunk 2]
```diff
@@ -12,11 +12,11 @@ This library takes API Blueprint AST and returns specific HTTP transactions (Req
 - Inherits parameters from parent Resource and Action sections
 - Expands URI templates
 - Warns on undefined URI query and path parameters
-- Validates URI parameteres types
-- Selects first request and first response if multiple request or responses are given in the 
-- Assigns path origin object pointing to the Blueprint AST 
-- Compiles [cannonical transaction path](#canonical-transaction-paths) as a unique identifier for each transaction
-- Assigns origin object pointing to the Blueprint AST [DEPRECATED]
+- Validates URI parameters types
```
Comment (kylef) [inline README.md]: Should both of these be plural? "parameters types" 

``` diff
- parameters types
+ parameter types
```

File: README.md [hunk 3]
```diff
@@ -41,12 +41,12 @@ Following is description is in a [MSON](https://github.com/apiaryio/mson) format
...
 - transaction (object)
     - name: `"Hello world! > Retrieve Message"` (string) Transaction identification name used for referencing
-    - path: `::Hello world!:Retreive Message:Example 1` (string) Transaction [cannonical path](#canonical-transaction-paths)
+    - path: `::Hello world!:Retreive Message:Example 1` (string) Transaction [canonical path](#canonical-transaction-paths)
 
     - request (object) Request compiled from blueprint
         - body `"Hello world!\n"` (string)
         - headers (object)
-        - uri `"/message"` (string) informative uri about the request
+        - uri `"/message"` (string) informative URI about the request
```
Comment (kylef) [inline README.md]: Perhaps:

``` diff
- URI about the request
+ URI for the request
```

**Comment by honzajavorek** [comment]
@kylef Thanks, comments addressed.

**Comment by kylef** [comment]
:+1:

**Pull request merged**